client/src/main.rs

```diff
@@ -1153,9 +1151,11 @@ async fn dep_workload(guest: &Arc<Guest>, ri: &mut RegionInfo) -> Result<()> {
         }
 
         guest.show_work()?;
-        println!("Loop:{} send a final flush and wait", my_count);
-        let mut flush_waiter = guest.flush()?;
-        flush_waiter.block_wait()?;
+        if random() && random() {
```

Comment:
The final flush is to help prevent the pause that we get when the last command is a write or read and we have to wait x seconds for the flush check to trigger.  Maybe take the rand out of that?

Reply:
done in 830187a